ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml

```diff
@@ -38,6 +38,23 @@ tests:
       commands: |
         set -euo pipefail
 
+        # FIXME: transfer into Dockerfile for ci
```

Comment:
Mind to ask one thing @dhiller, aren't the oc and kubectl already present in the shared folder (${SHARED_DIR}) ?

Reply:
No they aren't any more, that's one thing everyone now needs to provide on his own, AFAIU DPTP is working on a solution for this.

Reply:
ah, ok, so I know my current job with changes for multistage test steps is going to fail :)